blog/docs/articles/new_event_discovery_features.md

```diff
@@ -70,7 +70,8 @@ default     93774a924a741245a94313745d78e69f   dev.knative.sources.ping   /apis/
 ```
 #### Auto Event Type creation
 
-tbd
+Furthermore to improve the experience with `EventTypes` consumption and creation, there's a new experimental feature to automatically create `EventTypes` objects based on processed events on the broker ingress. Instead of manually creating them as yaml manifests along the application code that talks to the Broker API. This behaviour can be enabled by feature flag `eventtype-auto-creation` in `config-features` ConfigMap. For futher details and examples pleaase refer to [the documentation](https://knative.dev/docs/eventing/experimental-features/eventtype-auto-create/).
```

Comment:
I think @Cali0707 added this for 1.11 release also for the IMC, right?

Reply:
Yep, this should work for IMC now!

Reply:
@Cali0707 PTAL on the current update. Thanks!